test/fixtures/pubsub/main.tf

```diff
@@ -14,14 +14,10 @@
  * limitations under the License.
  */
 
-resource "random_id" "random_suffix" {
-  byte_length = 2
-}
-
 module "example" {
-  source     = "../../examples/simple"
+  source     = "../../../examples/simple"
   project_id = var.project_id
-  topic_name = "ci-int-topic-${random_id.random_suffix.hex}"
+  topic_name = var.topic_name
```

Comment:
If the `count` argument depends on the resource like `random_id`, the error happens.

Reply:
Oh that's a big issue, I will add a note above.